iree/samples/vision/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ iree_lit_test(
     iree_run_mnist_module_test
   TEST_FILE
     "mnist_test.txt"
-  DATA
+  TOOLS
```

Comment:
new failure in here?

```
FAIL: IREE :: samples/vision/mnist_test.txt (1 of 1)
******************** TEST 'IREE :: samples/vision/mnist_test.txt' FAILED ********************
Script:
--
: 'RUN: at line 1';   (iree-run-mnist-module) | FileCheck /tmpfs/src/github/iree/iree/samples/vision/mnist_test.txt
--
Exit Code: 2

Command Output (stderr):
--
../iree/tools/utils/image_util.c:44: NOT_FOUND; can't load image mnist_test.png; load image
FileCheck error: '<stdin>' is empty.
FileCheck command line:  FileCheck /tmpfs/src/github/iree/iree/samples/vision/mnist_test.txt
```

logs: https://source.cloud.google.com/results/invocations/29a1c095-42e7-4504-bb96-42cd58c88626/targets/iree%2Fgcp_ubuntu%2Fcmake%2Flinux%2Fx86-swiftshader%2Fpresubmit/log

Reply:
Yeah I think I have samples turned off in my local build, so I missed this. And it isn't covered by Bazel->CMake